build.sh

```diff
@@ -239,7 +239,7 @@ docker_commands () {
         build_tool_binary linux amd64 $cmd "tools"
     done
 
-    for cmd in slack; do
+    for cmd in slack influx-db; do
```

Comment:
This is a small thing, but we should probably make handlers a variable and loop through it where needed in build.sh.

Reply:
@cyphus I can probably do that in sh, but not quite sure how to do that for ps.